google/cloud/storage/internal/curl_wrappers_disable_sigpipe_handler_test.cc

```diff
@@ -33,12 +33,12 @@ TEST(CurlWrappers, SigpipeHandlerDisabledTest) {
 #if __has_feature(memory_sanitizer)
   // The memory sanitizer seems to intercept SIGPIPE, simply disable the test
   // in this case.
-  return;
+  GTEST_SKIP();
```

Comment:
Did you intend for all the `GTEST_SKIP()` stuff to be part of this PR?

Reply:
Yes, I was looking at where these functions were used and got carried away. I can split it if you want.

Reply:
Just checking.